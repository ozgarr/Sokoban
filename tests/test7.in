##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################

a4
0
b8
d4
d6
c2
0
c8
d8
d4
d6
a4
0
0
0
0
0
0
0
.
