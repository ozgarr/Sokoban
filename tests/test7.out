##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----+A@------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++--------@----#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++--------@----#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-@d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--@--++-------------#
#---#--c-----------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-@d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#--@--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#--@--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#--@--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--@d#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#-----++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#---d#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--+A@-------b----#
#-#-#-----++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--@d#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#-----++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#--d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#--c--++a-------b----#
#-#-#--@--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-@d-#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++-------------#
#---#--------------------#
##########################
##########################
#-#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a-------b----#
#-#-#--c--++--------@----#
#---#--------------------#
##########################
##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
##########################
#@#---#---#---#---#---#--#
#-#-#-#-#-#-#-#-#-#-#-#--#
#-#-#---#---#---#---#-d--#
#-#-#-####################
#-#-#--------------------#
#-#-###################--#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-####################-#
#-#-#--------------------#
#-#-#-####################
#-#-#--------------------#
#-#-#-----++a------------#
#-#-#--c--++--------b----#
#---#--------------------#
##########################
